hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -43,7 +43,7 @@ public void run(HookContext hookContext) {
 			    wholeQuery = LineageConstants.query;
 			    processId = "" + LineageConstants.processId;
 			    instanceExecId = "" + LineageConstants.instanceId;
-
+				LOGGER.debug("---------------query: "+ wholeQuery+" processid: "+processId+" iEid: "+instanceExecId);
 		    } else {
 //			    hookContext.getConf().getAllProperties().list(System.out);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)